hackathon_site/dashboard/frontend/src/components/general/Footer/Footer.test.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import Footer from "./Footer";
+
+describe("Footer", () => {
+    test("renders", () => {
+        const { asFragment } = render(<Footer />);
+        expect(asFragment()).toMatchSnapshot();
+    });
+
+    test("date", () => {
+        const fakeDate = new Date("2021-05-17");
+        const fakeYear = "2021";
```

Comment:
This doesn't do anything. If we were to change the year in `fakeDate` above, this variable would be wrong and the test would fail. Have the test below check for `fakeDate.getFullYear()`